### PR TITLE
Participant

### PR DESCRIPTION
**Pull request by Isylwin**

### CODE REVIEW

File: logic/src/main/nl/jft/logic/participant/Participant.java [hunk 1]
```diff
@@ -1,8 +1,17 @@
 package nl.jft.logic.participant;
 
 /**
+ * The {@code Participant} interface defines a participant that can partake in {@link nl.jft.logic.match.Match Matches} and {@link nl.jft.logic.tournament.Tournament Tournamements}.
+ * A {@code Participant} is stored in the database and thus should be //TODO add identifiable link when implemented.
```
Comment (TehMessiah) [inline logic/src/main/nl/jft/logic/participant/Participant.java]: INFO Complete the task associated to this TODO comment. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1135)

File: logic/src/main/nl/jft/logic/participant/Participant.java [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+ *
  * @author Lesley
+ * @author Oscar de Leeuw
  */
 public interface Participant {
```
Comment (TehMessiah) [inline logic/src/main/nl/jft/logic/participant/Participant.java]: CRITICAL Annotate the "Participant" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)

File: logic/src/main/nl/jft/logic/participant/impl/Team.java [hunk 3]
```diff
@@ -1,10 +1,42 @@
 package nl.jft.logic.participant.impl;
 
+import nl.jft.common.util.Arguments;
 import nl.jft.logic.participant.Participant;
 
 /**
+ * A {@code Team} represents a group of {@link Participant Participants} that form a team.
+ * Implements the {@code Participant} interface as a {@code Team} can partake in {@link nl.jft.logic.match.Match Matches} and {@link nl.jft.logic.tournament.Tournament Tournaments}.
+ * A {@code Team} is stored in the database and is thus //TODO add identifiable link.
```
Comment (TehMessiah) [inline logic/src/main/nl/jft/logic/participant/impl/Team.java]: INFO Complete the task associated to this TODO comment. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1135)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* CRITICAL 1 critical
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by Isylwin** [comment]
retest this please

**Pull request merged**